src/examples/can/feather-m4-can.go

```diff
@@ -0,0 +1,19 @@
+// +build feather_m4_can
+
+package main
+
+import (
+	"machine"
+)
+
+func init() {
+	// power on the CAN Transceiver
+	boost_en := machine.BOOST_EN
+	boost_en.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	boost_en.High()
```

Comment:
And what does this do?

Reply:
This operation is used to turn on the power of the CAN transceiver.
Since the on-board CAN transceiver requires 5V, the power supply can be controlled in consideration of LiPo drive.

https://learn.adafruit.com/adafruit-feather-m4-can-express/pinouts#can-bus-3078990-8

Reply:
Same as above comment, but adding `func (can *CAN) Boost(enable bool) error`

Reply:
This process is specific to feather-m4-can, so I don't think it is necessary for the CAN function. I added the above URL to the comments.